benchmarks/operator_benchmark/pt/configs.py

```diff
@@ -11,6 +11,11 @@ def remove_cuda(config_list):
     return [config for config in config_list if cuda_config not in config]
 
 
+def remove_cpu(config_list):
```

Comment:
Just curious: Why do you need to remove CPU device on those conv ops?  Because they are all on backward path?